### PR TITLE
Minor change bugs fixed on storage and marketplace pages

### PR DESCRIPTION
**Pull request by IslomJonMirzakbarov**

### CODE REVIEW

File: apps/web/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
```
Comment (JM066) [inline apps/web/.vscode/settings.json:1]: Can you remove this one?

Comment (IslomJonMirzakbarov) [reply apps/web/.vscode/settings.json:1]: yep, removed

File: apps/web/src/pages/Marketplace/MarketplaceGrid/MarketplaceCard/index.tsx [hunk 2]
```diff
@@ -9,14 +9,17 @@ interface ICollectionCard {
...
+
 function CollectionCard({ DataItem: { id, creator, title, collection, price } }: ICollectionCard) {
   const navigate = useNavigate();
 
   return (
     <div onClick={() => navigate(`/marketplace/${collection}/${id}`)} style={{ cursor: 'pointer' }}>
       <Card variant="primary" border={true}>
         <div className={styles.TopItems}>
-          <img className={styles.Image} src={`src/assets/images/NFTTiles/NFT_Tiles-${id}.jpg`} alt="NFT IMAGE" />
+          <img className={styles.Image} src={uri(id)} alt="NFT IMAGE" />
```
Comment (JM066) [inline apps/web/src/pages/Marketplace/MarketplaceGrid/MarketplaceCard/index.tsx:22]: Can you create a declaration file called declaration.d.ts in src and include it in the tsconfig and then import the images instead?

File: apps/web/src/pages/SingleItem/index.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ function SingleItem() {
       <RightSideLayer title="xxx">
         <div className={styles.ContentContainer}>
           <div className={styles.UpperContainer}>
-            <img className={styles.Image} src={`../../assets/images/NFTTiles/NFT_Tiles-${id}.jpg`} alt="NFT IMAGE" />
+            <img className={styles.Image} src={require(`../../assets/images/NFTTiles/NFT_Tiles-${id}.jpg`)} alt="NFT IMAGE" />
```
Comment (JM066) [inline apps/web/src/pages/SingleItem/index.tsx:58]: This one as well.

File: apps/web/src/pages/Marketplace/MarketplaceGrid/MarketplaceCard/index.tsx (hunk 2 above)
Comment (IslomJonMirzakbarov) [reply apps/web/src/pages/Marketplace/MarketplaceGrid/MarketplaceCard/index.tsx:22]: it is more realated to webpack configs than tsconfig

**Pull request closed without merging**